### PR TITLE
Install NASM 2.16 for Ubuntu Bionic base images

### PR DESCRIPTION
**Pull request by arjungm**
b/264574490

Change-Id: I566481b595a80c850a29dd421635577adbf39250

### CODE REVIEW

File: docker/linux/base/build/Dockerfile [hunk 1]
```diff
@@ -22,12 +22,19 @@ RUN apt update -qqy \
...
         binutils \
         bison \
-        nasm \
         ninja-build \
         pkgconf \
+        software-properties-common \
         unzip \
     && /opt/clean-after-apt.sh
 
+# === Install nasm 2.16 using backport from PPA
```
Comment (kaidokert) [inline docker/linux/base/build/Dockerfile:31]: The code seems correct but this should only go to old Ubuntu images, not unconditionally in base image

Comment (kaidokert) [reply docker/linux/base/build/Dockerfile:31]: E.g the file you want this in is probably https://github.com/youtube/cobalt/blob/main/docker/linux/gcc-6-3/Dockerfile

Comment (arjungm) [reply docker/linux/base/build/Dockerfile]: Ack, moved the files

File: docker/linux/clang-3-9/Dockerfile [hunk 2]
```diff
@@ -24,5 +24,12 @@ RUN apt update -qqy \
     && apt install -qqy --no-install-recommends git \
     && /opt/clean-after-apt.sh
 
+# === Install nasm 2.16 using backport from PPA
```
Comment (kaidokert) [inline docker/linux/clang-3-9/Dockerfile:27]: Looks better, but now you can consolidate this block with the above that already does the other PPA and cleanup - less clutter and faster build ?

Comment (arjungm) [reply docker/linux/clang-3-9/Dockerfile]: Ack, condensed the RUN directives

**Review by kaidokert — APPROVED** [review]
lgtm

**Pull request merged**